src/Authentication/Authentication/StartupScripts/ExportCustomCommands.ps1

```diff
@@ -0,0 +1,22 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+
+# Load custom commands
+$customScriptCommandDirItem = Get-Item $PSScriptRoot -ErrorAction Ignore
+if ( $customScriptCommandDirItem ) {
+    $customScriptCommandDir = join-path $customScriptCommandDirItem.FullName ../custom
+
+    Get-ChildItem $customScriptCommandDir -Filter *.ps1 -ErrorAction Stop | ForEach-Object {
+        . $_.FullName
+    }
+}
+
+# Export custom script commands without removing the
+# binary cmdlets. Custom script commands are functions,
+# the cmdlets are.. cmdlets. We must explicitly specify
+# both functions and cmdlets at export; if only one of
+# these classes is specified, nothing of the other
+# class will be exported.
+Export-ModuleMember -Function * -Cmdlet *
```

Comment:
Just an FYI Exporting module members like this will suppress currently exported cmdlet aliases such as `Connect-Graph`. The Pester tests to confirm this can be found [here](https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/fc50dfe002d11adfbe70fae1d8d6f12a8d8a24e5/src/Authentication/Authentication/test/Microsoft.Graph.Authentication.Tests.ps1#L58-L68). I've fixed this in my PR by adding [`Get-ModuleCmdlet`](https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/816/files#diff-1850068b7ea78c6849651f57381ce2283f70405350c22cd4f6ad75d7a632738fR16-R18.) and [`Get-ScriptCmdlet`](https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/816/files#diff-387044d693da97c2f19f907463d42ebc596aee909532d409ebc1615ff59213ccR11) helper cmdlets to aid in exporting functions, cmdlets, and aliases.

This will be fixed https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/816 when we merge the 2 PRs.

Reply:
> Other than the issue highlighted [here](https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/809/files#r687265838) (which has been fixed by PR #816), everything else looks good to me!

Awesome, thank you @peombwa! What are the next steps? It sounds like this is in your queue to merge and you'll be the person to handle merging this (along with the related PR's)?

Reply:
@adamedx, That's right. I'll merge the PR later today (to give others time to review it) then schedule everything for a 1.7.0 release somewhere next week.

@FehintolaObafemi, thank you for the excellent contribution!! I'm quite sure our customers will find this command valuable for their scripts.